v7/hierarchical_pages/README.md

```diff
@@ -1,46 +1,46 @@
 This plugin allows to translate paths by specifying paths in a hierarchy.
 
 Assume you have the following hierarchy of posts (default language English):
```

Comment:
Thanks. I thought it would work, though, since I used it in another plugin and thought it works. Now I notice that no, it didn't work either in the other one...